<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard</class>
 <widget class="QDialog" name="dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cyclist Info Calculator Spectacular</string>
  </property>
  <widget class="QLabel" name="lblRpm">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RPM</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCalories">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Calories/Hr</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSpeed">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>KPH</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRpms">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSpeed">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCalories">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
